### PR TITLE
Fix for logback

### PR DESCRIPTION
**Pull request by sbrossie**

### CODE REVIEW

File: logback.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
   -->
 
 <configuration>
+    <property name="LOGS_DIR" value="/var/lib/tomcat/logs" />
```
Comment (pierre) [inline logback.xml]: IIRC we said we don't want this (we'd rather change `LOGS_DIR` to default to `.` instead of `./logs`)?

Comment (sbrossie) [reply logback.xml]: I thought we also said, i tried and it dod not work :-)

Comment (sbrossie) [reply logback.xml]: Now, you make me doubt, i guess i am gonna have to try again...

Comment (sbrossie) [reply logback.xml]: Ack - just removed it.

**Pull request merged**